examples/streaming/wordcount-java/src/test/java/org/apache/gearpump/streaming/examples/wordcountjava/WordCountSpecJava.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.gearpump.streaming.examples.wordcountjava;
+
+import org.apache.gearpump.cluster.ClusterConfig;
+import org.apache.gearpump.streaming.examples.wordcountjava.dsl.WordCount;
+import org.apache.log4j.Logger;
+import org.junit.Test;
+
+/**
+ * Created by Buddhi on 7/14/2017.
+ */
+public class WordCountSpecJava {
```

Comment:
Is this used anywhere ? It's a simple wrapper around `org.apache.gearpump.streaming.examples.wordcountjava.dsl.WordCount`